### ISSUE TITLE
allow large metadata thumbnail to appear when clicked on

### ISSUE THREAD

**Issue by takurkowski** [ua-snap/snap-www#49]
http://athena.snap.uaf.edu:8080/geonetwork/srv/en/metadata.show.embedded?id=21

I'm going to have someone produce all these thumbnails, but in the meantime, it'd be nice to have the below work correctly.

when accessed through geonetwork, when you click on the "small" thumbnail, the "large" one appears, allowing users to see more details of our data.  When accessing the meta record via the map tool or data page, this does not work.  Afer talking with Dustin, it looks as though the large thumbnail is accessed through javascript that is not being interpreted or passed correctly.

**Comment by brucecrevensten** [comment]
For the 'click to enlarge' to work, GeoNetwork's got source code wrapped up
around the metadata that isn't present when we do a direct request for the
metadata.  I think I see a way to make a request work, and worst-case, we
can vacuum up whatever javascript it's using on GeoNetwork and plorp it
into the SNAP web site code.

On Wed, Apr 4, 2012 at 11:01 AM, Tom Kurkowski <
reply@reply.github.com

> wrote:
> 
> http://athena.snap.uaf.edu:8080/geonetwork/srv/en/metadata.show.embedded?id=21
> 
> I'm going to have someone produce all these thumbnails, but in the
> meantime, it'd be nice to have the below work correctly.
> 
> when accessed through geonetwork, when you click on the "small" thumbnail,
> the "large" one appears, allowing users to see more details of our data.
>  When accessing the meta record via the map tool or data page, this does
> not work.  Afer talking with Dustin, it looks as though the large thumbnail
> is accessed through javascript that is not being interpreted or passed
> correctly.
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/ua-snap/snap-www/issues/49

## 

*
*
*Bruce Crevensten, Web Programmer
*
Scenarios Network for Alaska & Arctic Planning
3352 College Road, 2nd Floor Denali Building
Fairbanks, AK 99709
Phone: 907-474-7134
Fax: 907-474-7151
www.snap.uaf.edu
becrevensten@alaska.edu

**Comment by brucecrevensten** [comment]
Rats.  My idea didn't work and this gets bound up in cross-domain issues -- since we're embedding as an iframe, we can't easily inject JS into the necessary context, so either we need to make a request that gets us a properly-formed + embedded page, or fetch + rewrite the data in some way.

**Comment by brucecrevensten** [comment]
I took a different stab at this (injecting scripts into the iframe) and it just feels like a terrible solution.

What we want is to see the metadata and/or the image and present it nicely.  I found [some XSL transforms](http://www.ngdc.noaa.gov/metadata/published/xsl/iso2faq/) that look like exactly what we'd want to use for this -- we can modify those as required to yield better markup / use of fields.

I'm going to suggest that if this change is a priority, that we improve the underlying metadata + access to the metadata.
